qt/python/mantidqt/widgets/memorywidget/test/test_memorypresenter.py

```diff
@@ -42,8 +48,10 @@ def test_presenter(self):
         self.assertEqual(self.presenter.view.set_value.call_count, 1)
         self.assertEqual(self.presenter.view.set_bar_color.call_count, 1)
 
-        self.presenter.update_memory_usage()
-        self.assertEqual(self.presenter.view.set_value.call_count, 2)
+    def test_memory_usage_is_updated_based_on_a_constant(self):
```

Comment:
I don't quite follow what the test name means here? Tests that depend on timings make me nervous. Can we atleast import the `TIME_INTERVAL_MEMORY_USAGE_UPDATE` constant from the module and add something to it?

Reply:
Tests that the thread is doing what it is supposed to do, the variable is now used!